packages/taquito-sapling/src/sapling-transaction-viewer.ts

```diff
@@ -1,23 +1,275 @@
-import * as sapling from '@airgap/sapling-wasm'
+import * as sapling from '@airgap/sapling-wasm';
+import BigNumber from 'bignumber.js';
+import { hex2buf, mergebuf } from '@taquito/utils';
+import { CommitmentsAndCiphertexts, RpcClient, RpcClientInterface } from '@taquito/rpc';
+import blake from 'blakejs';
+import { openSecretBox } from '@stablelib/nacl';
```

Comment:
could you give a brief explanation on what openSecretBox does?

Reply:
openSecretBox allows decrypting an encrypted message